controllers/locationController.ts

```diff
@@ -15,23 +18,100 @@ class LocationController implements Controller{
         this.router.get(this.path, this.getStatsForLocation);
     }
 
+
     getStatsForLocation = async(req: express.Request, res: express.Response) => {
         try {
-            const location: Location = {
-                City : '',
-                Territory : 'MANITOBA',
-                Country : req?.query?.location?.toString() ?? 'canada',
-                XCoord : 0.00,
-                YCoord : 0.00
+            if (req.query && req.query.longitude && req.query.latitude) {
```

Comment:
Probably should be extracted and handled by a mapper. This will need additional discussion and investigation.

Reply:
Sounds good, we can talk about this tomorrow.